mailpoet/assets/js/src/date.ts

```diff
@@ -21,8 +21,8 @@ export const MailPoetDate: {
   convertFormat: (format: string) => string;
   isInFuture: (dateString: string, currentTime: MomentInput) => boolean;
   isInPast: (dateString: string, currentTime: MomentInput) => boolean;
-  gmtToSiteTimestamp: (utcTimeStamp: string) => string;
-  siteToGmtTimestamp: (siteTimeStamp: string) => string;
+  datetimeString: (utcTimeStamp: string) => string;
+  toGmtDatetimeString: (siteTimeStamp: string) => string;
```

Comment:
Probably forgotten, I suppose here the parameters should be `date: MomentInput`.